packages/terra-table/src/subcomponents/Row.module.scss

```diff
@@ -3,6 +3,10 @@
 
 
 :local {
+  .striped-table-row:nth-child(even) {
+    background-color: var(--terra-table-row-striped-background-color, #f0f0f0);
+  }
+
```

Comment:
For some reason, this was causing issues with row selection. I had to move this up to the same level as `.row` because CSS specificity would always use the striped table row style instead of the selected style.